src/flyte/_image.py

```diff
@@ -789,13 +791,13 @@ def with_requirements(
         return new_image
 
     def with_pip_packages(
-        self,
-        *packages: str,
-        index_url: Optional[str] = None,
-        extra_index_urls: Union[str, List[str], Tuple[str, ...], None] = None,
-        pre: bool = False,
-        extra_args: Optional[str] = None,
-        secret_mounts: Optional[SecretRequest] = None,
+            self,
```

Comment:
why are these whitespace changes coming in now?  did we change the ruff version?